vspace/src/main/java/edu/asu/diging/vspace/core/services/IModuleManager.java

```diff
@@ -1,11 +1,24 @@
 package edu.asu.diging.vspace.core.services;
 
+import java.util.List;
+
+import edu.asu.diging.vspace.core.exception.ModuleDoesNotExistException;
 import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.services.impl.CreationReturnValue;
 
 public interface IModuleManager {
 
     IModule storeModule(IModule module);
 
     IModule getModule(String id);
+    
+    ISlide createSlide(String id, String title, String description) throws ModuleDoesNotExistException;
```

Comment:
this is the moduleId, right? id is very confusing, sounds like the id of the new slides needs to be passed.